Example/AppDelegate+CarPlay.swift

```diff
@@ -72,7 +72,16 @@ extension AppDelegate: CarPlayManagerDelegate {
             searchTemplate.delegate = carPlaySearchController
             let searchButton = carPlaySearchController.searchTemplateButton(searchTemplate: searchTemplate, interfaceController: interfaceController, traitCollection: traitCollection)
             return [searchButton]
-        case .navigating, .previewing:
+        case .navigating:
+            let muteTitle = "Mute"
```

Comment:
Why are these no longer localizable? You may have to the `bundle` parameter or move the localizable strings into the CarPlay framework.

Reply:
@frederoni strings are not localized in the `Example` app. This is not the `CarPlay framework` PR #1943 It is an unrelated pull request.